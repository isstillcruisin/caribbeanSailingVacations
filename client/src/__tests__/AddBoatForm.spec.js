import React from "react";
import { create } from "react-test-renderer";
import AddBoatForm from '../components/AddBoatForm';

describe("AddBoatForm component", () => {
  test("Matches the snapshot", () => {
    const form = create(<AddBoatForm 
      boatName='test boat'
      crewBio='test bio'
      maxPassengers='10'
      year='1999'
      manufacture='2000 feet test manufacture'
      imgs={[]}
    />);
    expect(form.toJSON()).toMatchSnapshot();
  });
});