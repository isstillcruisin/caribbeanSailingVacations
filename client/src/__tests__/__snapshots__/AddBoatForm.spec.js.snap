// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddBoatForm component Matches the snapshot 1`] = `
<form>
  <div
    className="form-group add-boat-form"
  >
    <label
      htmlFor="boat-name"
    >
      <strong>
        Boat Name
      </strong>
    </label>
    <input
      className="form-control"
      id="boat-name"
      name="boatName"
      placeholder="name of the boat..."
      required={true}
      type="text"
      value="test boat"
    />
    <label
      htmlFor="description"
    >
      <strong>
        Crew Bio
      </strong>
    </label>
    <input
      className="form-control"
      id="crewBio"
      name="crewBio"
      placeholder="about the crew.."
      required={true}
      type="textArea"
      value="test bio"
    />
    <label
      htmlFor="maxPassengers"
    >
      <strong>
        Passenger Capacity
      </strong>
    </label>
    <input
      className="form-control"
      id="maxPassengers"
      name="maxPassengers"
      placeholder="4-30"
      required={true}
      type="number"
      value="10"
    />
    <label
      htmlFor="year"
    >
      <strong>
        Year made
      </strong>
    </label>
    <input
      className="form-control"
      id="year"
      name="year"
      placeholder="2009"
      required={true}
      type="number"
      value="1999"
    />
    <label
      htmlFor="manufacture"
    >
      <strong>
        Manufacturer of Boat
      </strong>
    </label>
    <input
      className="form-control"
      id="manufacture"
      name="manufacture"
      placeholder="Boat size etc..."
      required={true}
      type="text"
      value="2000 feet test manufacture"
    />
    <label
      htmlFor="image-url"
    >
      <strong>
        Boat Images
      </strong>
    </label>
    <div
      className="boat-image-dropzone"
      onBlur={[Function]}
      onClick={[Function]}
      onDragEnter={[Function]}
      onDragLeave={[Function]}
      onDragOver={[Function]}
      onDrop={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      tabIndex={0}
    >
      <input
        accept="image/*"
        autoComplete="off"
        multiple={true}
        onChange={[Function]}
        onClick={[Function]}
        style={
          Object {
            "display": "none",
          }
        }
        tabIndex={-1}
        type="file"
      />
      <p>
        Try dropping some files here, or click to select files to upload.
      </p>
    </div>
    <div>
      <div>
        <div />
      </div>
    </div>
  </div>
  <div
    className="pull-right"
  >
    <button
      className="btn btn-lg btn-danger"
      type="submit"
    >
      Submit
    </button>
  </div>
</form>
`;
